.gitignore

```diff
@@ -50,3 +50,4 @@
 /llvm
 /src/r_drawersasm.obj
 /src/r_drawersasm.o
+/build_cmake
```

Comment:
What's the point of this change? I think we have more than enough of them already.